docs/en/sql-reference/functions/other-functions.md

```diff
@@ -3688,3 +3720,118 @@ Result:
 ```response
 {'version':'1','serial_number':'2D9071D64530052D48308473922C7ADAFA85D6C5','signature_algo':'sha256WithRSAEncryption','issuer':'/CN=marsnet.local CA','not_before':'May  7 17:01:21 2024 GMT','not_after':'May  7 17:01:21 2025 GMT','subject':'/CN=chnode1','pkey_algo':'rsaEncryption'}
 ```
+
+## lowCardinalityIndices
+
+For each row in the current [block](../../development/architecture.md/#block-block), returns the index of the value in the dictionary of unique values for columns of [LowCardinality](../data-types/lowcardinality.md) type. 
+
+:::note
+The first unique value encountered in the block is enumerated from 1.
+:::
+
+**Syntax**
+
+```sql
+lowCardinalityIndices(col)
+```
+
+**Arguments**
+
+- `col` — a low cardinality column. [LowCardinality](../data-types/lowcardinality.md).
+
+**Returned value**
+
+- returns the index of the value in the dictionary of unique values, for each row in the current block. [UInt64](../data-types/int-uint.md).
```

Comment:
`- The position of the value in the dictionary of the current part. [UInt64](../data-types/int-uint.md).`

(Notes: We should try to reduce confusion due to different terminology. "Position" is a more fitting name for the result than "index".)